internal/stack/clients.go

```diff
@@ -158,5 +170,10 @@ func FindCACertificate(profile *profile.Profile) (string, error) {
 		caCertPath = profileConfig.CACertificatePath
 	}
 
+	// Avoid returning an empty certificate path, fallback to the default path.
+	if caCertPath == "" {
+		caCertPath = profile.Path(CACertificateFile)
+	}
```

Comment:
In case of Serverless projects that are not required to set the CA certificate path, would this cause errors ?

Reply:
It shouldn't, this CA certificate is only additive. Leaving this empty mounts an empty file in the dockerfile.

Reply:
Ok :+1: 

I was wondering if it could affect in case it is used with a Serverless project that was created out of `elastic-package`. And the user sets the environment variables and it does not set the environment variable related to the CA to run for instance `elastic-package install`.

Reply:
I haven't found regressions in the serverless provider with these changes.